crates/cargo-wdk/tests/common.rs

```diff
@@ -14,10 +14,18 @@ use fs4::fs_std::FileExt;
 pub fn set_crt_static_flag() {
     if let Ok(rustflags) = std::env::var("RUSTFLAGS") {
         let updated_rust_flags = format!("{rustflags} -C target-feature=+crt-static");
-        std::env::set_var("RUSTFLAGS", updated_rust_flags);
+        // SAFETY: env::set_var is always safe on Windows according to documentation,
```

Comment:
technically someone could be building this on non-windows (ex. for rustdoc). an idea is to have a safe set_var function that just panics if called from non-windows. it looks like we're only using set_var in tests and in cargo-make which is probably fine to limit to just windows